community-nightlies/rpm.sh

```diff
@@ -54,6 +54,37 @@ pgdg_check ()
   fi
 }
 
+epel_check()
+{
+  echo "Checking for EPEL repositories..."
+
+  epel_release_check=`rpm -qa | grep -qw epel-release`
+  if [ "$?" == "0" ]; then
+    echo "Detected EPEL repoitories"
+  else
+    echo -n "Installing epel-release repo... "
+
+    if [ "${os}" = "centos" ]; then
+      yum install -d0 -e0 -y epel-release &> /dev/null
+    elif [ ${os} = "rhel" ] || [ ${os} = "redhatenterpriseserver" ]; then
+      if [ ${dist} = "7" ]; then
+        yum install -d0 -e0 -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${dist}.noarch.rpm &> /dev/null
+      elif [ ${dist} = "8" ]; then
+        dnf install -d0 -e0 -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${dist}.noarch.rpm &> /dev/null
+      fi
+    fi
+
+    epel_release_check=`rpm -qa | grep -qw epel-release`
```

Comment:
we should remove -w flag (whole word search) because since we are installing from an address and a specific version, package name also includes version number as below. I got it from ubi8 by executing  rpm -qa
rpm -qa | grep  "epel-release"
epel-release-7-13.noarch

Reply:
`-w` whole world search accepts `-` characters as word boundaries. So it should be ok if the package name is `epel-release-7-13.noarch`

```
❯ echo "epel-release\nepel-release-7-13.noarch\nepel-releasewordwitoutspace" | grep -w epel-release
epel-release
epel-release-7-13.noarch
```

Reply:
![image](https://user-images.githubusercontent.com/1596331/110120355-feb56300-7dcd-11eb-9e69-d4aecd3b4fb7.png)

Reply:
I tried it and actually can not find it

Reply:
`-q` flag means quiet and therefore you do not see any output. You can try `grep -w epel-release`.

PS: We had a chat and writing this comment for documentation purposes.

Reply:
Thanks approving PR